lib/langfuse_sdk.ex

```diff
@@ -68,6 +105,19 @@ defmodule LangfuseSdk do
     LangfuseSdk.Ingestor.ingest_payload(score_event)
   end
 
+  @doc """
+  Get a list of traces.
+
+  ## Params
+
+  Check the [official API docs](https://api.reference.langfuse.com/#get-/api/public/traces) for the available params.
+
+  ## Example
+
+      LangfuseSdk.list_traces()
```

Comment:
```suggestion
      LangfuseSdk.list_traces(%{page: 1,  limit: 10})
```

Does it require string keys or atoms are good?